src/components/FreeOffer/FreeOffer.js

```diff
@@ -17,18 +16,20 @@ import {
   ButtonWrapperStyled,
   ErrorMessageStyled
 } from './FreeOfferStyled';
+import { fetchPaymentWithoutDetails } from '../../redux/paymentMethodsSlice';
```

Comment:
we have absolute import set to the `src` folder so we can simplify this to `from 'redux/paymentMethodsSlice';`